runtime/vm/segment.c

```diff
@@ -240,6 +240,17 @@ allocateMemoryForSegment(J9JavaVM *javaVM,J9MemorySegment *segment, J9PortVmemPa
 		 */
 		Assert_VM_true(J9_ARE_NO_BITS_SET(segment->type, MEMORY_TYPE_VIRTUAL) || J9_ARE_ANY_BITS_SET(segment->type, ~MEMORY_TYPE_VIRTUAL));
 
+		/* Memory segments can be marked as MEMORY_TYPE_DISCLAIMABLE_TO_FILE
+		 * to indicate the intent of allocating memory backed-up by a file.
+		 * This information is passed to the omr port library by setting the
+		 * flag OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN.
+		 * Note that when setting when the OMRPORT_VMEM_MEMORY_MODE_SHARE_TMP_FILE_OPEN bit,
+		 * we also have to set the OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN bit.
```

Comment:
Small typo coming from the original code: "when" is repeated.

Reply:
Fixed in force push.